src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -773,13 +792,17 @@ export function useSwapInputsController({
             });
 
             const inputAssetBalance = internalSelectedInputAsset.value?.balance.amount || '0';
-            const updatedSliderPosition = clamp(
-              Number(divWorklet(current.values.inputAmount, inputAssetBalance)) * SLIDER_WIDTH,
-              0,
-              SLIDER_WIDTH
-            );
 
-            sliderXPosition.value = withSpring(updatedSliderPosition, snappySpringConfig);
+            if (equalWorklet(inputAssetBalance, 0)) {
+              sliderXPosition.value = withSpring(0, snappySpringConfig);
```

Comment:
I did a similar fix on my branch for the div by zero crash. We can keep this one and I'll rebase after yours is in